Rakefile

```diff
@@ -12,6 +12,7 @@ XCODE_WORKSPACE = 'Simplenote.xcworkspace'
 XCODE_SCHEME = 'Simplenote'
 XCODE_CONFIGURATION = 'Debug'
 PROJECT_DIR = __dir__
+LOCAL_PATH = "vendor/bundle"
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/simplenote-macos/blob/3ac2567ac42e7b095ea9c63f66dba316e3bd2de2/Rakefile#L15"></span>Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.</td>
    </tr>
  </tbody>
</table>